src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -546,8 +546,22 @@ export const isDisabilityPtsd = disability => {
   });
 };
 
+export const hasRatedDisabilities = formData =>
+  formData?.ratedDisabilities?.length > 0;
```

Comment:
@Mottie I can change this to just be `formData?.ratedDisabilities?.length` since 0 is already a falsy value, but I thought this more clearly represented what we are checking here. Let me know if you think I should change this.

Reply:
I'm reviewing this PR now.. I like this comparison, it makes it very clear.